ddl/ddl_api.go

```diff
@@ -257,7 +382,8 @@ func (d *ddl) ModifySchemaSetTiFlashReplica(ctx sessionctx.Context, stmt *ast.Al
 		failStmt = fmt.Sprintf("(including table %v)", oneFail)
 	}
 	msg := fmt.Sprintf("In total %v tables: %v succeed, %v failed%v, %v skipped", total, succ, fail, failStmt, skip)
-	ctx.GetSessionVars().StmtCtx.SetMessage(msg)
+	sctx.GetSessionVars().StmtCtx.SetMessage(msg)
+	logutil.BgLogger().Info("finish batch add TiFlash replica", zap.Int64("schemaID", dbInfo.ID))
```

Comment:
Could we remove this log?

Reply:
We can use this log to determine whether the process is finished. 
Though we can induce that from response to MySQL, but the client may have disconnected.